src/Sarif.Viewer.VisualStudio/Resources.resx

```diff
@@ -359,4 +359,22 @@
   <data name="OpenExternalUri_DialogTitle" xml:space="preserve">
     <value>Open Hyperlink</value>
   </data>
+  <data name="ConfirmSourceFileDialog_BrowseFileButton" xml:space="preserve">
+    <value>Browse To Alternate Location</value>
+  </data>
+  <data name="ConfirmSourceFileDialog_Message" xml:space="preserve">
+    <value>This log has embedded file content for selected item. We found file in your solution/enlistment that matches relevant file paths but which appears to be the wrong version (hash mismatch). What do you want to do?</value>
```

Comment:
>file [](start = 74, length = 4)

the filee

Reply:
updated according to Michael's feedback

---
In reply to: [601270789](https://github.com/microsoft/sarif-visualstudio-extension/pull/384#discussion_r601270789) [](ancestors = 601270789)